src/GovUk.Education.ExploreEducationStatistics.Common/ViewModels/PaginatedListViewModel.cs

```diff
@@ -0,0 +1,70 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.ViewModels
+{
```

Comment:
For new classes, we're adopting the non-indented version of the namespace syntax.  So the majority of the class doesn't need to be indented.